src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -208,5 +209,55 @@ public string Value
         ///   Returns <see cref="JsonValueKind.String"/>
         /// </summary>
         public override JsonValueKind ValueKind { get => JsonValueKind.String; }
+
+        /// <summary>
+        ///   Converts the text value of this instance, which should encode binary data as base-64 digits, to an equivalent 8-bit unsigned <see cref="byte"/> array.
+        ///   The return value indicates wether the conversion succeeded.
+        /// </summary>
+        /// <param name="value">
+        ///   When this method returns, contains the <see cref="byte"/> array equivalent of the text contained in this instance,
+        ///   if the conversion succeeded.
+        /// </param>
+        /// <returns>
+        ///   <see langword="true"/> if text was converted successfully; othwerwise returns <see langword="false"/>.
+        /// </returns>
+        internal bool TryGetBytesFromBase64(out byte[] value)
+        {
+            if (_value.Length < 1)
+            {
+                value = default;
+                return false;
+            }
+
+            // we decode string -> byte, so the resulting length will
+            // be /4 * 3 - padding. To be on the safe side, keep padding in slice later
```

Comment:
`in` -> `and` ? Otherwise, I don't know how to interpret the comment
```suggestion
            // be /4 * 3 - padding. To be on the safe side, keep padding and slice later
```